packages/studio-web/src/app/upload/upload.component.html

```diff
@@ -84,18 +85,23 @@ <h2 i18n="Title for record box" class="subtitle">Record your voice here</h2>
                         Stop
                       </button>
                     </div>
+                  <p *ngIf="!recording && audioControl.value !== null" i18n="playback invitation">Please Play your audio to make sure it is OK, and save it for reuse if it is.</p>
```

Comment:
Similarly here, I'd rather this was a temporary toast message than markup on the page.